### PR TITLE
Fix timeout, stdio options, and switch dep to tmp.

### PR DESCRIPTION
**Pull request by krisselden**
Fix timeout for DevToolsActivePort file to match behavior of chromedriver https://chromium.googlesource.com/chromium/src/+/6fd4390daea50497e5ead4583829e2e8f9b215b2/chrome/test/chromedriver/chrome_launcher.cc#445

Add stdio option to spawnBrowser so you can "ignore" stdio (defaults to current behavior of inheriting the process stdio).

Fix issues with last update to execa (changed buffer option and it creates a promise for the spawned process).

### CODE REVIEW

File: lib/spawn-browser.ts [hunk 1]
```diff
@@ -72,14 +79,23 @@ class BrowserProcess implements IBrowserProcess {
...
-      // disable buffer, pipe to stdout
-      maxBuffer: null as any,
-    });
-    child.stdout.pipe(process.stdout);
-    child.stderr.pipe(process.stderr);
-    child.on("error", (err: Error) => (this.lastError = err));
+      // disable buffer, pipe or drain
+      buffer: false,
+      stdio,
+    } as any);
```
Comment (asakusuma) [inline lib/spawn-browser.ts]: Can we not type this?

File: lib/spawn-browser.ts [hunk 2]
```diff
@@ -72,14 +79,23 @@ class BrowserProcess implements IBrowserProcess {
...
+    // child is now a thenable for the duration of the promise
+    // ensure we handle its error or we will get an
+    // unhandled rejection warning
+    child.catch(err => (this.lastError = err));
```
Comment (asakusuma) [inline lib/spawn-browser.ts]: Would it be useful to keep a list of errors if the child has multiple errors? Not sure if that's even possible.

File: lib/spawn-browser.ts (hunk 1 above)
Comment (krisselden) [reply lib/spawn-browser.ts]: it is typed, but the types on DefinitelyTyped are stale. It used to be maxBuffer: null was how you disabled buffer but it changed in 1.0

File: lib/spawn-browser.ts (hunk 2 above)
Comment (krisselden) [reply lib/spawn-browser.ts]: The process exits on error, execa `child` is a promise like for the lifetime of the child process.

**Pull request merged**